global/autoload/save_file/save_file.gd

```diff
@@ -244,7 +244,7 @@ func _export_save_data() -> Dictionary:
 	save_data["npc_events"] = npc_events
 	save_data["enemy"] = enemy
 	save_data["metadata"] = metadata
-	save_data["LOCALE"] = LOCALE
+	save_data["locale"] = locale
```

Comment:
note to self: check if this breaks stuff, because it's a change in data structure